### DISCUSSION TITLE
How to center a Table

### DISCUSSION THREAD

**Discussion by mperrando** [buildo/bento-design-system#306] [Q&A] [answered]
Hi all.

I need to have a table centred on the page.

I placed it inside a full width Box and a centred Stack.

The problem is, when the page is narrow, the table does not scroll fine: it scrolls correctly only to the right, while, to the left, a part of table is not reachable.

Can you help me to sort this thing out? Thanks.

[Playroom](https://playroom.bento.buildo.io/#?code=N4Igxg9gJgpiBcIA8AhCAPABAdwJZQBcALAXgB0QAzAVwBtaLMIA3GAJ0tomwA1yQAhtQIQKAPjIA7TJiQBlAgLABrTAGcADkpglgABgC%2BmAbVwBzSfzAxJBduKkyZSACoCARrRiOnMqAMVdAG0fXxlgULCZSQEAWxh4TAoAQViBAC8IaQFJKHUIeKZidkxIWxsCTBFMNOUYKqJcNRx8epsIajMiCgAaSKjIals2AE9EiOkosNwyxIBJMrnJSgg2NIJcVj7Jqac7dAJEigBVOV7%2BsINt3Zk1RQJqNXGLqNoPGFojkGSwDdZznY3SBcNhfMxsGA2AE3TBXF5OZgmagJTATGHRajxNgzABqSJRAEY9HpruiZll5otlqt1psYKTdnDAVFsDB3GpcHZnsypkQIZQvkQCAQNPAAPRi7BSgB0cQyWWlkFi0PRb3cHy%2BABlcJJlCrGQypiZ2AQnqj4b5yZJKVkAOoCNiSHVmQ27NUapIge2O536qZMmHgjqiz0AcTYwcwBL9UVgXjsPw2FPNPNe70%2BnoAIh8YHYY1MrTbJABhLhqekWpxZAAKELUZoAFABKTAkMSogM3TtRbthNE3GLxL6hiAQMxefO%2BQbDMYp9GYQuYBZZJYrNYBOmuqb7Q6e06Tpy9qLuYQia1z1Xpr4AJW4mAATAffModVAvgQ2DlNA6Kk%2BnI12AdMAiFnCgNGxNJRj-GQazrRsWzbDst18I8wgEDQNFMMANyyM0iQAfWJElKzuAJHm5ed3QzChqyEcsoGg0oIBBL5PGRP9UN8RFaGRCj0UkTF2FxfFEgJABWYjU2mWYlypNdaS2StOKcVl2U5FF%2BxhPkYAFT0hRFcVJRlMxR3HGBFQKRiqK1HU9RAZDDwcmQg2oEMKHDSNH3sys41zGBE1wZNNJuaysxzPNvKky0ZOXEsywrKKq0kWsYHrRJm1bdtgGUmRlNQgBdAwLmBTFJDUYJ4UUTwYGLZjSulHcG2CqIiBgARYDYTUr09AA5OJvEimElGsetVi%2BQcBocgwmwcqqvFqnjYkkBqYAOW1OSIWKmsrVr2vYLr1WokBaqGD8Rj-YbUrUMbPWnM6-ytasIA5JNzwobEugIA9ptmjx5rqpbpQE2J1TYbbEt2jqDo9FIMKwnCyousARuu0FPXQzCZgRtQ-245EADEaQCOw0YbPGYAQ9slgIWhpR6zFQcJ9cCAbCgoRAJtpXkgIyfxGbKxMcw3pAD6hW%2B-mpLmmqAeW4HQfW4gtuasJIf27qKDxHjJoF5Grpuihydx-EmfWEn0vJyml1sWn6ZB9gTZ5tnJAoTnuZZi2nOMUwLC%2BUWvsG-0Jd2KWFvq0xdXBrS2qh9WvTZF7tcSy7RrRihVIT8Xfuq0PAatSOblVzrY%2BSLw2FNJGUf1wRS-LgOeyD7c-ulxbluA3ANHz3ZC%2Bho6FDInG66NXWU6%2BUiHgHz3BZ926KnsQeUIbqIQ5l6UrRQU8sk73lo7Vw6vgC3CK711OQF8hNfkC5357Q73hesco2D-KAmibuRVgIFBZw-ZEpsXpxCsiGKCQkwkBij7ioYBoC0DoGAfZEAeBCBEDUAgIIhUgA)

**Comment by gabro** [comment, marked as answer]
Hi, I'm not sure of what's going on with `Stack` there, I will need to take a closer look.

In the meantime, you should be able to work around this by replacing `Stack` with `Box`, like so:

https://playroom.bento.buildo.io/#?code=N4Igxg9gJgpiBcIA8AhCAPABFAlgZwAcAbAQwE8BeAHRADMiZ0bMB3HKAFwAtq6BXIkWYQAbjABO9CCwAavEnw4RmAKz54OOWmQDCEAHYcYh3mGNHxNAHxV9mTEgAqJAEYNb9%2B1BIcSFYADaHp72wMEh9vokALYw8Jg0AILRJABeBpgk%2BlCYeBCxmBDcEpiQhuaYSpgpANYwlVz4rOz1xhB8AOZcNAA04RGQfIbiZPFhdhEhOGXxAJJls-q0EOIpmmJ9E5P2Rugc8TQAqgDKvf0hAL6b27m%2BHOpj5xGkLjBEByCJYOswZ1vbkCIKw%2BHXEMGMfxuVye9hEJCIfDimHGN0ifFi4mmADV4Yj4gBGAAMhOuqOmBjmCyWKzWOA2MMw0P%2BIRYMBceBwRkezIiXDBtA%2BXA4HAI8AA9GKWFKAHQxNIGaWQaKQ1EvN4fAAyOH0NRVkyZN3hEg4eG5qMw5P0lIMAHUSOJ9NqOqSbmr3gkQHaHU69REDdtQe1RR6AOLiIOYfG%2BkKwBhGL6aCnIhn2N0fAAibxgRmjEUt1v0OiBeBgLu2BgACmC8KbMAAKACUmAoVmR-v1Zfs7c8KO2UViHxDEAgHXcIE7nkGw1GyZ5UxmmHmBkWy1WPjppZTlUY%2Bw9J1zXYn9hciiUVtn5rTHoAStJMAAmA%2BeGraqAfDjiLKEe3mJ-2RoSPaYBcDONAEJiKQjH%2BhT6FWMA1vEjbNq2wDdpcR6ZAQxDTOuBi1kSAD6xIkgyGg%2BA8F6qq46oehWCgllA0GAsCHpuIiT5oZ4cIIkivY3Po6ISNiuJIviACsJFzp4%2BaLlSq60vSUmcfYrLspyvFbnyMACh6QoiuKkoyh0w6jjAir5NBV40FqOocRhgZ8MGNBhhGj7jgysbZjACY4EmfHbFZICZnGvzuVJ9gyUuhbFpu4UwXBCH1k2LZthhnHdgAuhc-SAui%2Bh4P4QQ8r4bgwHoCLRPo0q7Bwdb%2BSEXAwCQsDiBq1HujQAByMShRhJBgGYNYsTQ-a9U8FwNhOJUMOVeXVTuNqclwUV1QyjXNRIbWvB1IB6EMH5kE%2B-WDXk4gfFOB1PpaFYQByibnjQmJdBwuYTVNrgzRAFVVQJ0SvOIq1SetLVbTRSRYUQOH3XgR0DfBp0fCQENQ75%2BVPtxiIAGI0j4FiIRjMDJa2iwcEQ0qdei-3Y2utU0BCIANtK8k%2BHWBOTQy8I4B0D0gE9QqvezxUfWVX1zb9-2LdwK31Z4wObe1Hw4jxsMncNIAE%2BjInU2seP1mzyGLoYZMU39EjayzdP6DQjPM7VbN9ZD3MfHzL1hfqguTNNIvfdKkM6oDNxy61CsejabJ3WNUnHfDauqRHAvvaVs2VdKloB9sQegztiQMOIJoqzHZ0ekaecw27foexEXvJ1VwE4AQ6eTJnIc0McdzqAXQ1FzQZH3GXDtczzZjlJY5eXJXITV6LKeWigp4GI3vJNSDLefN8qP9xzcNdx8nnxuvBhHY7Q-mBIT64HgwvHCsHAoDOH6IhOb39FlwRijY%2BhIGKaDoB-44gGwTgXA8AIACFlIAA
